client/landing/gutenboarding/index.tsx

```diff
@@ -134,26 +134,24 @@ function waitForSelectedSite(): Promise< Site | undefined > {
 	} ).finally( unsubscribe );
 }
 /**
- * If available-designs-config.json has been updated, replace the cached list
- * of designs with the updated designs
+ * If the list of available designs (stored in the `@automattic/domain-picker` package)
```

Comment:
s/domain/design/

Reply:
Well spotted. Not sure what happened to my brain when adding design picker comments in this file!

Fixed in https://github.com/Automattic/wp-calypso/pull/51633/commits/f923bdbf308024aa413ed864687b6237abb8dddb